plugins/backend/ttrss/ttrssAPI.vala

```diff
@@ -20,11 +20,26 @@ public class FeedReader.ttrssAPI : GLib.Object {
 	private string m_ttrss_sessionid;
 	private uint64 m_ttrss_apilevel;
 	private Json.Parser m_parser;
+	private string? m_iconDir = null;
```

Comment:
This is not the local icon directory, but the one on the server. Apparently the location of the icons are configurable for tt-rss.